EIPS/eip-1521.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 1521
+title: Insurance Policies as ERC721 Tokens
```

Comment:
The discussion URL uses "Standard for Insurance Policies as ERC-721 Non fungible tokens". Please make the two titles more closer?

Reply:
done